source/beamline/CLS/CLSAdvancedScalerChannelDetector.cpp

```diff
@@ -115,17 +343,17 @@ void CLSAdvancedScalerChannelDetector::onScalerScanningChanged(bool isScanning){
 void CLSAdvancedScalerChannelDetector::onReadingChanged(){
 
 	if(readMode_ == AMDetectorDefinitions::ContinuousRead){
-		QVector<int> allIntReadings = scaler_->reading();
-		int enabledChannelCount = scaler_->enabledChannelCount();
-		int totalDataCount = allIntReadings.at(0)/enabledChannelCount;
-
-		continuousData_.clear();
-		continuousSize_ = totalDataCount;
-		for(int x = 0; x < totalDataCount; x++)
-			continuousData_.append(allIntReadings.at(x*enabledChannelCount + channelIndex_ + 1));
+//		QVector<int> allIntReadings = scaler_->reading();
```

Comment:
Yep, as a reminder that this needs to be handled when we have trigger/dwell working.